app/models/poly_metadatum.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# A polymetadatum can be assigned to any metadatable record.
```

Comment:
This makes it sound to me like `metadatable` is the name of an actual model (would have me searching for one called 'metadatable' until I noticed the 'polymorphic' keyword further down and remembered what it meant). Could we add a bit more detail, something like:

A polymetadatum is a key value pair store. It is built to be generic so that it can be associated with multiple different models. It can be linked to any model that has the reverse association set up.

Reply:
Added a better description with instructions how to implement on other models.